amundsen_application/static/js/ducks/user/reducer.ts

```diff
@@ -53,9 +73,28 @@ export default function reducer(state: UserReducerState = initialState, action:
       return { ...state, loggedInUser: action.payload };
     case GetUser.ACTION:
     case GetUser.FAILURE:
-      return { ...state, profileUser: defaultUser };
+      state.profile.user= defaultUser;
+      return { ...state };
```

Comment:
Much prefer this here:
```
{
  ...state,
  profile: {
    ...state.profile,
    user: defaultUser,
  }
}
```
 The state is supposed to be immutable and I wouldn't want this to temp others into recklessly starting to use `state.blah = foo` elsewhere. Also sure the above is more lines of code, but this is also one of the reason the redux state is supposed to either be kept as flat as possible or use nested reducers so that each reducer is pretty much just handling one level of the state.
https://github.com/reduxjs/redux/issues/432